leetcode/src/5.c

```diff
@@ -0,0 +1,34 @@
+char * longestPalindrome(char * s){
+	char *result = NULL;
+	int strSize = 0;
```

Comment:
You can use func `strlen` instead of for-loop:
`int strSize = strlen(s);`

Reply:
Yeah, that's probably better. Slipped my mind.